libs/entities/area.py

```diff
@@ -25,6 +26,45 @@ def __init__(self, config_section: dict, section_title: str, send_email_enabled:
             self.should_send_slack_notifications = False
         self.load_occupancy_rules()
 
+    @classmethod
+    def set_global_area(cls, is_email_enabled, is_slack_enabled, config_dir, area_logs_dir, cameras_list):
+        config_path = os.path.join(config_dir, "ALL.json")
+        json_content = {
+            "global_area_all": {
+                "ViolationThreshold": -1,
```

Comment:
I think 0 is a better default for `ViolationThreshold`, `NotifyEveryMinutes` and `OccupancyThreshold`

Reply:
Changed!